json/signup.js

```diff
@@ -0,0 +1,76 @@
+import {testUser, emailValidCheck, passwordValidCheck, addError, removeError, passwordToggle} from "./global.js";
+const emailInput = document.querySelector("#email");
+const emailError = document.querySelector(".email-error");
+const passwordInput = document.querySelector("#password");
+const passwordError = document.querySelector(".password-error");
+const passwordCheckInput = document.querySelector("#password-check");
+const passwordCheckError = document.querySelector(".password-check-error");
+const confirmBtn = document.querySelector('#confirm-button');
+const eyeBtn = document.querySelector('.eye-btn');
+const anotherEyeBtn = document.querySelector('.eye-btn2');
+
+
+/*입력받은 email 값을 판별하고 조건에 맞는 에러메시지를 출력하는 함수*/
+function validEmailInput(e) {
+  if (e.target.value === '') {
+    addError(emailInput, emailError, '이메일을 입력해 주세요.');
```

Comment:
에러메시지들이 signup.js와 signin.js에서 반복사용되니 상수로 따로 빼서 반복사용하면 좋겠네요.

```suggestion
  /** error-message.js */
  const EMAIL_REQURIED = '이메일을 입력해 주세요.';

  /** signup.js */
  addError(emailInput, emailError, EMAIL_REQURIED);
```